src/services/session/session.hooks.js

```diff
@@ -1,14 +1,15 @@
 const { authenticate } = require('@feathersjs/authentication').hooks;
+const permission = require('../../hooks/permission');
 
 module.exports = {
   before: {
     all: [authenticate('jwt')],
     find: [],
     get: [],
-    create: [],
-    update: [],
-    patch: [],
-    remove: [],
+    create: [permission({ roles: ['coach', 'manager'] })],
+    update: [permission({ roles: ['coach'] })],
```

Comment:
```suggestion
    update: [permission({ roles: ['coach', 'manager'] })],
```
Manager should be able to update sessions